rtc_service/rtc_service.py

```diff
@@ -141,7 +150,10 @@ def run(self):
         """
         while True:
             self.publish_time()
```

Comment:
Why using a global variable and not returning True or False from publish_time ?
It seems a bit cleaner to me.
```python
if  self.publish_time():
        sleep(self.period)
else:
        sleep(self.retry_period)
```

Reply:
You are right, I will correct it